src/daft-micropartition/src/python.rs

```diff
@@ -658,17 +658,15 @@ impl PyMicroPartition {
 
     #[staticmethod]
     pub fn _from_unloaded_table_state(
-        schema_bytes: &PyBytes,
-        loading_scan_task_bytes: &PyBytes,
-        metadata_bytes: &PyBytes,
-        statistics_bytes: &PyBytes,
+        schema_bytes: &[u8],
```

Comment:
this would be a good use of bound api to the underlying byte slices. Let's verify that's the case.

Reply:
See my reply to `_from_serialized`